modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js

```diff
@@ -195,6 +198,50 @@ class SegmentEdit extends Component {
 	_isQueryEmpty = (contributors) =>
 		contributors.every((contributor) => !contributor.query);
 
+	/**
+	 * Checks if every item inside criteriaMap > items array has empry/falsy value in its value property.
+	 * @return {boolean} True if a non trythy values is found.
+	 */
+	_handleQueryEmptyValues = (contributors) => {
```

Comment:
I would maybe change the name of this function because we are not handling de values, we are checking if they exists. Maybe something like: checkQueryEmptyValues or hasQueryEmptyValues would be better. What do you think?

Reply:
I agree. This fn used to have some side effects modifying the state, but now that is pure I think is a good idea to change it 😃

Reply:
As it is returning a boolean I think `_queryHasEmptyValues` is a proper name